components/cors-mgt/org.wso2.carbon.identity.cors.mgt.core/src/main/java/org/wso2/carbon/identity/cors/mgt/core/internal/impl/CORSManagementServiceImpl.java

```diff
@@ -161,7 +166,8 @@ public void deleteCORSOrigins(String applicationId, List<String> originIds, Stri
             if (!existingCORSOrigins.stream().map(CORSOrigin::getId).collect(Collectors.toList()).contains(originId)) {
                 // CORS origin is not registered for the application.
                 if (log.isDebugEnabled()) {
-                    log.debug(String.format(ERROR_CODE_ORIGIN_NOT_PRESENT.getMessage(), tenantDomain, originId));
```

Comment:
handle null for applicationBasicInfo

Reply:
Handled in `getApplicationBasicInfo` private method.

Reply:
Handled in `getApplicationBasicInfo` private method.